src/main/resources/templates/crowdLogin.html

```diff
@@ -45,20 +45,28 @@
 
 <!-- ensure IE does not fly -->
 <script type="text/javascript" >
-	let isIE = false;
-    if(/MSIE/i['test'](navigator['userAgent'])==true||/rv/i['test'](navigator['userAgent'])==true||/Edge/i['test'](navigator['userAgent'])==true){
-       alert("ODS Provision application does not support Internet Explorer! Please use Google Chrome");
-       isIE = true;
-    }
+
+    var browserNotAllowed = false;
     
+    if(navigator.userAgent.indexOf("Firefox") != -1 )
+    {
+        alert("ODS Provision application does not support Firefox! Please use Google Chrome");
+        browserNotAllowed = true;
+    }
+    else if((navigator.userAgent.indexOf("MSIE") != -1 ) || (!!document.documentMode == true )) //IF IE > 10
+    {
+        alert("ODS Provision application does not support Internet Explorer! Please use Google Chrome or Firefox");
```

Comment:
The comment should be 

`alert("ODS Provision application does not support Internet Explorer! Please use Google Chrome");`